components/dashboard/dashboard.go

```diff
@@ -139,6 +139,28 @@ func (d *Dashboard) ReconcileComponent(owner metav1.Object, cli client.Client, s
 		if err != nil {
 			return fmt.Errorf("failed to set dashboard ISV from %s: %v", PathISVSM, err)
 		}
+
+		// ConsoleLink handling
+		consoleRoute := &routev1.Route{}
+		err = cli.Get(context.TODO(), client.ObjectKey{Name: "console", Namespace: "openshift-console"}, consoleRoute)
```

Comment:
minor nitpick: I would suggest creating constants for the console name and namespace to make easier to maintain in the future.

Reply:
updated